cmd/util/format.go

```diff
@@ -3,15 +3,50 @@ package cmdutil
 
 import (
 	"fmt"
+	"io"
+	"os"
 	"time"
 
 	"github.com/fatih/color"
 )
 
+// Stdout represents Stdout
+var Stdout io.Writer = os.Stdout
+
+// Stderr represents Stderr
+var Stderr io.Writer = os.Stderr
+
+// ColoredStderr represents a color supporting writer for Stderr
+var ColoredStderr io.Writer = color.Error
```

Comment:
Reason for this is so the tests don't have to depend on the `color.Error` package